config.template.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "https://raw.githubusercontent.com/Perlten/Architectural-Lens/master/config.schema.json",
+    "$schema": "config.schema.json",
```

Comment:
are we always going to find the `config.schema.json` file in the current folder from where we're running the code?

Reply:
will this work when installed with pip?

Reply:
Do you have any resources for local pip development? We are a bit unsure of how to test the changes made to the pip package locally

Reply:
The new changes should fix this issue :)